src/components/transport_manager/include/transport_manager/transport_manager_default.h

```diff
@@ -63,6 +64,18 @@ class TransportManagerDefault : public TransportManagerImpl {
    */
   virtual ~TransportManagerDefault();
 
+#if defined(BUILD_TESTS)
+  void set_ta_bluetooth(transport_adapter::TransportAdapter* ta_bluetooth);
+  void set_ta_tcp(transport_adapter::TransportAdapter* ta_tcp);
+  void set_ta_usb(transport_adapter::TransportAdapter* ta_usb);
+  void set_ta_cloud(transport_adapter::TransportAdapter* ta_cloud);
+#endif  // BUILD_TESTS
+
+ private:
+  transport_adapter::TransportAdapter* ta_bluetooth_;
+  transport_adapter::TransportAdapter* ta_tcp_;
+  transport_adapter::TransportAdapter* ta_usb_;
```

Comment:
@SNiukalov Do we really need to use raw pointer here?

Reply:
@ZhdanovP If we are not going to change al inheritance chain and methods - than yes. We just moved raw pointers from local function variables to class members

Reply:
@ZhdanovP 
Fixed in:https://github.com/smartdevicelink/sdl_core/pull/3134/commits/f653a628ed63e5a95fa2564a8700c348b865b70b